src/Redux/api.tsx

```diff
@@ -353,6 +353,13 @@ const routes = {
     path: "/api/v1/facility/{facility_external_id}/asset_location/{external_id}/",
     method: "PATCH",
   },
+  deleteFacilityAssetLocation: {
+    path: "/api/v1/facility/{facility_external_id}/asset_location/{external_id}/",
+    method: "DELETE",
+    TRes: Type<null | {
+      detail?: string;
+    }>(),
```

Comment:
`TRes` does not define the type of error or responses for non-2xx responses.

And 2xx responses for DELETE requests do not have a response body.

```suggestion
    TRes: Type<Record<string, never>>(),
```